freqtrade/optimize/hyperopt.py

```diff
@@ -293,21 +293,6 @@ def get_results(self, results) -> str:
             self.hyperopt_table_header = 2
         return output
 
-    @staticmethod
-    def print_results_explanation(results, total_epochs, highlight_best: bool,
```

Comment:
You might want to remove `get_results()` too? 

It doesn't seem to be used either...

Reply:
Sure, nice catch